baseplate/context/memcache/__init__.py

```diff
@@ -107,31 +107,147 @@ def __init__(self, context_name, server_span, pooled_client):
+            span.set_tag("key", key)
+            return self.pooled_client.get(key, **kwargs)
+
+    def get_many(self, keys):
+        with self._get_span("get_many") as span:
+            span.set_tag("key_count", len(keys))
+            return self.pooled_client.get_many(keys)
+
+    def gets(self, key, **kwargs):
+        with self._get_span("gets") as span:
+            span.set_tag("key", key)
+            return self.pooled_client.gets(key, **kwargs)
+
+    def gets_many(self, keys):
+        with self._get_span("gets_many") as span:
+            span.set_tag("key_count", len(keys))
+            return self.pooled_client.gets_many(keys)
+
+    def delete(self, key, noreply=None):
+        with self._get_span("delete") as span:
+            span.set_tag("key", key)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.delete(key, noreply=noreply)
+
+    def delete_many(self, keys, noreply=None):
+        with self._get_span("delete_many") as span:
+            span.set_tag("key_count", len(keys))
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.delete_many(keys, noreply=noreply)
+
+    def add(self, key, value, expire=0, noreply=None):
+        with self._get_span("add") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("expire", expire)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.add(key, value, expire=expire, noreply=noreply)
+
+    def incr(self, key, value, noreply=False):
+        with self._get_span("incr") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.incr(key, value, noreply=noreply)
+
+    def decr(self, key, value, noreply=False):
+        with self._get_span("decr") as span:
+            span.set_tag("key", key)
+            span.set_tag("value", value)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.decr(key, value, noreply=noreply)
+
+    def touch(self, key, expire=0, noreply=None):
+        with self._get_span("touch") as span:
+            span.set_tag("key", key)
+            span.set_tag("expire", expire)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.touch(key, expire=expire, noreply=noreply)
+
+    def stats(self, *args):
+        with self._get_span("stats"):
+            return self.pooled_client.stats(*args)
+
+    def flush_all(self, delay=0, noreply=None):
+        with self._get_span("flush_all") as span:
+            span.set_tag("delay", delay)
+            span.set_tag("noreply", noreply)
+            return self.pooled_client.flush_all(delay=delay, noreply=noreply)
+
+    def quit(self):
+        with self._get_span("quit"):
+            return self.pooled_client.quit()
+
+    def _get_span(self, method_name):
```

Comment:
nit: this feels more like `_make_span` than `_get_span` since we're creating a new one each time